tools/ports/emdawnwebgpu.py

```diff
@@ -132,17 +144,37 @@
 package zip).
 """
 
-TAG = 'v20250807.221415'
+import sys
+
+if __name__ == '__main__':
+    print('Please see documentation inside this file for details on how to use this port.')
+    sys.exit(1)
+
+_VERSION = 'v20250926.144300'
 
-EXTERNAL_PORT = f'https://github.com/google/dawn/releases/download/{TAG}/emdawnwebgpu_pkg-{TAG}.zip'
-SHA512 = 'ab9f3af2536ef3a29c20bb9c69f45b5ee512b8e33fb559f8d0bf4529cd2c11e2fbfb919c3d936e3b32af0e92bd710af71a1700776b5e56c99297cfbc3b73ceec'
+# Remote-specific port information
+
+# - Where to download the port
+EXTERNAL_PORT = f'https://github.com/google/dawn/releases/download/{_VERSION}/emdawnwebgpu_pkg-{_VERSION}.zip'
+# - Hash to verify the download integrity
+SHA512 = 'a186cf7f33266c9dfeca7d99ffac769a91b2129e34054f1e857cd82e8b033896da34cf758088bbdeeb128aa713df9953851f83ba6d677c438a929d245a789948'
+# - Path of the port inside the zip file
 PORT_FILE = 'emdawnwebgpu_pkg/emdawnwebgpu.port.py'
 
-# Port information (required)
+# General port information
 
 # - Visible in emcc --show-ports and emcc --use-port=emdawnwebgpu:help
 LICENSE = "Some files: BSD 3-Clause License. Other files: Emscripten's license (available under both MIT License and University of Illinois/NCSA Open Source License)"
 
 # - Visible in emcc --use-port=emdawnwebgpu:help
 DESCRIPTION = "Emdawnwebgpu implements webgpu.h on WebGPU, replacing -sUSE_WEBGPU. **For info on usage and filing feedback, see link below.**"
-URL = 'https://dawn.googlesource.com/dawn/+/80062b708e44aa4d8c48e555ed0cc801396069f6/src/emdawnwebgpu/pkg/README.md'
+URL = 'https://dawn.googlesource.com/dawn/+/faa7054b5b65c3ce3774151952a68aa7668aa20b/src/emdawnwebgpu/pkg/README.md'
+
+
+# Emscripten <4.0.10 won't notice EXTERNAL_PORT and will try to use this.
+def get(ports, settings, shared):
+    raise Exception('Remote ports require Emscripten 4.0.10+.')
+
+
+def clear(ports, settings, shared):
```

Comment:
Can / should we update the port as a separate PR that we can land first?

Reply:
I'm fine either way, just did it in one PR because I needed the CI checks to run. Opened #25425